libs/design/notification/src/notification/notification.component.ts

```diff
@@ -55,10 +58,17 @@ export class DaffNotificationComponent
   implements DaffPrefixable, DaffStatusable {
   faTimes = faTimes;
 
+  @ContentChild(DaffPrefixDirective) _prefix: DaffPrefixDirective;
+
+  @ContentChild(DaffNotificationActionsDirective) _actions: DaffNotificationActionsDirective;
```

Comment:
This should be `private`

Reply:
Woops. Looks like the template uses this and can't be private.

Reply:
can't be private